### DISCUSSION TITLE
MPV设置音量时的显示

### DISCUSSION THREAD

**Discussion by ccccxxx** [hooke007/mpv_PlayKit#10] [Q&A 问答] [answered]
你好！
我使用MPV时，感觉设置音量时出现的提示与滑杆显示过大，影响观感。
实际使用时仅查看OSC进度条上的音量提示就足够了。

我尝试关闭OSC（使用Thumbnailer_OSC）显示后调节音量，依然会出现提示与滑杆。

请问有对应的设置可以关闭这项显示吗？或在哪个脚本里对其修改？

使用环境Win10 64位，使用脚本：
playlistmanager.lua
Thumbnailer.lua
Thumbnailer_OSC.lua
Thumbnailer_Worker.lua

**Comment by hooke007** [comment, marked as answer]
操作快捷键时的提示属于osd的一部分
input.conf 中相关指令前置 no-osd 即可，扩展阅读手册 05
`no-osd add volume 1`

另，osd的全体样式修改见mpv.conf的OSD部分

**Reply by Master-Panda** [reply]
作者你好，我用了如下的命令，WHEEL_UP    no-osd add volume 1，原来的提示不见了，但是我发现一个问题，就是uosc的音量控制条在播放视频的时候似乎只有鼠标靠近它，它才会自动浮现，在其他的地方调节音量时，音量控制台不显示，请问这该怎么设置

**Reply by hooke007** [reply]
你看看原本怎么写的
https://github.com/hooke007/MPV_lazy/blob/9713164e5bd2342c55c15902c690fd8095393aa0/portable_config/input_uosc.conf#L23-L24

**Reply by Master-Panda** [reply]
> 你看看原本怎么写的
> 
> https://github.com/hooke007/MPV_lazy/blob/9713164e5bd2342c55c15902c690fd8095393aa0/portable_config/input_uosc.conf#L23-L24

明白了，感谢感谢，之前用的自己写的
